### DISCUSSION TITLE
404: NOT_FOUND

### DISCUSSION THREAD

**Discussion by tomhusband** [tteck/Proxmox#3248] [General] [answered]
I get:

404: NOT_FOUND
Code: DEPLOYMENT_NOT_FOUND
ID: lhr1::52zhp-1719044979638-a1d12a59516a

when accessing https://helper-scripts.com/scripts. Is the site down?

**Comment by OddSquirrel** [comment]
Working fine for me. Maybe just a short hiccup.

**Comment by tteck** [comment, marked as answer]
Drop the `/scripts` or clear browser cache
